Unify the style of the TWC articles (#2178)
ppy/osu-wiki: wiki/Tournaments/TWC/2015/en.md

---
tags:

> - TWC2015

Reviewer: ```suggestion
- TWC 2015
- TWC2015
```

---
# osu! Taiko World Cup 2015

![TWC 2015 logo](img/logo.png)

The **osu! Taiko World Cup 2015** (***TWC 2015***) was a country-based osu!taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was the fifth installment of the osu!taiko World Cup.

## Tournament Schedule

| Event | Timestamp |
| --: | :-- |
| Registration Phase | 2015-01-12/2015-02-22 |
| Live Drawings | 2015-02-28 (14:00 UTC) |
| Group Stage | 2015-03-07/2015-03-08 |
| Round of 16 | 2015-03-14/2015-03-14 |
| Quarterfinals | 2015-03-21/2015-03-22 |
| Semifinals | 2015-03-28/2015-03-29 |
| Finals | 2015-04-04/2015-04-05 |
| Grand Finals | 2015-04-12 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter tag, unique profile badge, "Taiko Champion" user title, osu! merchandise |

Reviewer 2: ```suggestion
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | 6 months of osu!supporter tag, unique profile badge, "Taiko Champion" user title for one year, osu! merchandise |
```

- TWC 2015
- TWC2015
---
# osu! Taiko World Cup 2015

![TWC 2015 logo](img/logo.png)

> The **osu! Taiko World Cup 2015** (***TWC 2015***) was a country-based Taiko tournament hosted by the [osu! staff](/wiki/People/The_Team). It was the 5th installment of the osu!taiko World Cup.

Reviewer 2: ```suggestion
The **osu! Taiko World Cup 2015** (***TWC 2015***) was a country-based Taiko tournament hosted by the [osu! team](/wiki/People/The_Team). It was the fifth installment of the osu!taiko World Cup.
```

| ![][flag_SE] | **Sweden** | **[Bnyuu](https://osu.ppy.sh/users/2290294)**, [boat](https://osu.ppy.sh/users/276074), [Tanaka](https://osu.ppy.sh/users/1897905), [WilleGO97](https://osu.ppy.sh/users/3880756) |
| ![][flag_TW] | **Taiwan** | **[-\[ ix Ishida xi \]-](https://osu.ppy.sh/users/242910)**, [jordan\_sur](https://osu.ppy.sh/users/2960947), [zx\_baka\_0502442](https://osu.ppy.sh/users/457766), [howar5041](https://osu.ppy.sh/users/584411), [monkeydluffy3u4](https://osu.ppy.sh/users/2277798), [mliencheng](https://osu.ppy.sh/users/586659) |
| ![][flag_UA] | **Ukraine** | **[Anny](https://osu.ppy.sh/users/249094)**, [Aka](https://osu.ppy.sh/users/1307553), [gavnono](https://osu.ppy.sh/users/1625581) |
| ![][flag_US] | **United States** | **[Two Fruit Cakes](https://osu.ppy.sh/users/437259)**, [FLANKs](https://osu.ppy.sh/users/2398168), [OzzyOzrock](https://osu.ppy.sh/users/465153), [Shyguy](https://osu.ppy.sh/users/178038), [Kuro](https://osu.ppy.sh/users/1735031), [goatlife](https://osu.ppy.sh/users/4408784) |

> ------------------------------------------------------------------------

Reviewer 3: This article had info about groups, could be better to point those in a table below the participants?, i think it was ordered like Unseeded, low, mid, high and top, so it should be from top to unseed.

## Podium

This competition has come to an end and resulted in the following podium:

| Placing | Team(s) |
| :-: | :-- |
| ![Gold Crown](/wiki/shared/GCrown.png "1st place") | ![][flag_JP] Japan |
| ![Silver Crown](/wiki/shared/SCrown.png "2nd place") | ![][flag_TW] Taiwan |
| ![Bronze Crown](/wiki/shared/BCrown.png "3rd place") | ![][flag_FR] France |

> ![TWC 2015 Bracket](img/brackets.jpg)

Reviewer 2: Also change image name.
```suggestion
![TWC 2015 Bracket](img/bracket.jpg)
```

- TWC 2015
- TWC2015
---
# osu! Taiko World Cup 2015

> ![TWC 2015 logo](img/logo.png)

Reviewer 2: ```suggestion
![TWC 2015 Logo](img/logo.png)
```
